### ISSUE TITLE
Shelly Plus 1 does not connect to WiFi after the partition manager / migration start 

### ISSUE THREAD

**Issue by ok11** [tasmota/mgos32-to-tasmota32#61]
Hey all,

observing the following: 
- after unlocking the bootloader (the restart and wifi connect was successful) and starting the migration from the partition manager, the device does not seem to connect to the wifi
- the tasmota ap is running, but 192.168.4.1 does not bring up the web ui
- http commands are accepted, but do not work properly
  - for example, `http://192.168.4.1/wi?s1=ssid&p1=pwd&save=` returns
```
File Not Found

URI: /wi
Method: GET
Arguments: 3
 s1: ssid
 p1: pwd
 save: 
```

So, the device is not bricked, but also does not run as expected. What should I do now?

T&R, Oleg

**Comment by Jason2866** [comment]
Recover device by serial flashing Tasmota.

**Comment by ok11** [comment]
ok. I have a couple of other Plus 1s to brick (struck through) flash, shall I consider something specific? like shelly firmware downgrade to a specific version? Or like not going for ota reflash at all?

**Comment by Jason2866** [comment]
Bad Internet connection. In general the convert process does work well.

**Comment by ok11** [comment]
> Bad Internet connection. In general the convert process does work well.

Ah ok. Just to rephrase what you said to make sure I get it right: the latest tasmota release was not able to be downloaded, so the migration has stuck?

**Comment by Jason2866** [comment]
I don't say this was the case, it could be. A short power Interrupt will do harm too.

**Comment by ok11** [comment]
Sorry, to say that, but the same just happened to the second device. Wifi has been stable, no power interrupts. Have been waiting for like 10 minutes, now the tasmota AP has become active as in the first case. I can wait longer, but somehow I have a feeling that it won't work. Surely, I can do serial flashing, but it is like an indication that either I do something consistently wrong, or the migration process does not work as expected. The device is pretty warm.

**Comment by Jason2866** [comment]
Can't say what is going wrong here. My test Shelly device converts without an issue.

**Comment by ok11** [comment]
Oh. From AP I'm able to come to the safeboot. It seems the same issue as in https://github.com/arendst/Tasmota/discussions/19414

**Comment by Jason2866** [comment]
Provide the Tasmota firmware via local firmware upload (by file) when in safeboot mode.

**Comment by ok11** [comment]
> Provide the Tasmota firmware via local firmware upload (by file) when in safeboot mode.

Not sure how, it offers only "Console", no "Firmware Upgrade"

**Comment by Jason2866** [comment]
You should see with `http://192.168.4.1/up?`

**Comment by Caleb79** [comment]
I've a similar issue with this device and Tasmota.

Flashing was running smooth but as soon as I use auto-config or upload the template it takes about one minute and the device is going to standard settings incl. starting the AP mode again.

I tried to upload the latest tasmota firmware again (file-upload) but the device told me: wrong signature

Is there anything I can do?

**Comment by Jason2866** [comment]
> I tried to upload the latest tasmota firmware again (file-upload) but the device told me: wrong signature

Tasmota does not have a signature check

**Comment by Caleb79** [comment]
This what I get, trying to update.

<img width="553" alt="Bildschirmfoto 2024-05-07 um 14 24 17" src="https://github.com/tasmota/mgos32-to-tasmota32/assets/53936849/b5f5e68b-2c8e-4923-8e25-9c49beaf9fd1">

**Comment by Caleb79** [comment]
I could check the console before it falls back to defaults.
<img width="2047" alt="Bildschirmfoto 2024-05-07 um 15 39 07" src="https://github.com/tasmota/mgos32-to-tasmota32/assets/53936849/750ba7fa-236d-469b-a37c-9fd1365ee3e9">

**Comment by Jason2866** [comment]
The file you try to use is corrupt.

**Comment by Caleb79** [comment]
I used the latest release from this repository here and downloaded it a couple of times.
Even the OTA worked once but also no change on that.

**Comment by Jason2866** [comment]
If Tasmota is running, you can't and need to load anything from here.
For updates Tasmota needs to be used.

**Comment by Caleb79** [comment]
Okay, That explains my first update issue.
But nevertheless, even with a download from Tasmota (which worked very well - thanks for that), after changing the configuration, the device falls back to factory settings - again and again.

To be honest, I was thinking, that I might be using the wrong device ;) but no, it's the 1-plus.

A bit frustrating.

**Comment by ChristianWittke** [comment]
I can confirm the same situation. 
Did all the steps and now my shelly not usuable anymore.
First I updated it to the latest shelly version.
Then I put the latest tasmota zip to go for tasmota.
Tasmota Wifi was started and I added my wifi to it, I went for the mentioned steps, i.e. auto configuration and afterwards I started the Partition Wizard and to do Safeboot Conversion. After magic happens as written in the UI the shelly is starting now, but not connecting to my Wifi, but opening a Wifi which I can connect to, but the UI is not loading.
